src/Compilers/VisualBasic/Test/CommandLine/ErrorLoggerTests.vb

```diff
@@ -0,0 +1,202 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Globalization
+Imports System.IO
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.CodeAnalysis.Test.Utilities.SharedResourceHelpers
+Imports Microsoft.CodeAnalysis.VisualBasic.UnitTests
+Imports Xunit
+
+Imports Microsoft.CodeAnalysis.CommonDiagnosticAnalyzers
+Imports Microsoft.CodeAnalysis.DiagnosticExtensions
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.CommandLine.UnitTests
+
+    Public Class ErrorLoggerTests
+        Inherits BasicTestBase
+
+        Private Shared ReadOnly libDirectory As String = Environment.GetEnvironmentVariable("LIB")
```

Comment:
In general I'd prefer we avoid the pattern of initializing a static with an environment variable.  Our tests are intended to run in parallel in `AppDomain` boundaries and this type of initialization works against that.